src/signals/incident/components/form/DescriptionInput/DescriptionInput.js

```diff
@@ -24,7 +14,15 @@ const DescriptionInput = ({
     [value, meta.maxLength]
   );
 
-  const handleBlur = useCallback(event => get(event, meta, parent), [meta, parent]);
+  const handleBlur = useCallback(event => {
+    const {
+      getClassification,
+      updateIncident,
+      incidentContainer: { usePredictions },
+    } = parent?.meta;
+    if (usePredictions && event.target.value) getClassification(event.target.value);
```

Comment:
Personally, I'd go for readable code and change this line in
```javascript
if (usePredictions && event.target.value) {
  getClassification(event.target.value);
}
```

Reply:
agree!